src/main/scala/org/broadinstitute/dsde/workbench/sam/service/ManagedGroupService.scala

```diff
@@ -127,38 +131,57 @@ class ManagedGroupService(private val resourceService: ResourceService, private
     val resourceAndPolicyName = ResourceAndPolicyName(Resource(ManagedGroupService.managedGroupTypeName, resourceId), policyName)
     resourceService.removeSubjectFromPolicy(resourceAndPolicyName, subject)
   }
+
+  def requestAccess(resourceId: ResourceId, requesterUserId: WorkbenchUserId): Future[Unit] = {
+    val resourceAndPolicyName = ResourceAndPolicyName(Resource(ManagedGroupService.managedGroupTypeName, resourceId), ManagedGroupService.adminPolicyName)
+    accessPolicyDAO.listFlattenedPolicyUsers(resourceAndPolicyName).map { users =>
+      val notifications = users.map { recipientUserId =>
+        Notifications.GroupAccessRequestNotification(recipientUserId, WorkbenchGroupName(resourceId.value).value, users, requesterUserId)
+      }
+
+      cloudExtensions.fireAndForgetNotifications(notifications)
+    }
+  }
 }
 
 object ManagedGroupService {
   val managedGroupTypeName = ResourceTypeName("managed-group")
   val groupNameRe = "^[A-z0-9_-]+$".r
   private val memberValue = "member"
   private val adminValue = "admin"
+  private val adminNotifierValue = "admin-notifier"
 
   type ManagedGroupPolicyName = AccessPolicyName with AllowedManagedGroupPolicyName
   // In lieu of an Enumeration, this trait is being used to ensure that we can only have these policies in a Managed Group
   sealed trait AllowedManagedGroupPolicyName
   val adminPolicyName: ManagedGroupPolicyName = new AccessPolicyName(adminValue) with AllowedManagedGroupPolicyName
   val memberPolicyName: ManagedGroupPolicyName = new AccessPolicyName(memberValue) with AllowedManagedGroupPolicyName
+  val adminNotifierPolicyName: ManagedGroupPolicyName = new AccessPolicyName(adminNotifierValue) with AllowedManagedGroupPolicyName
 
   def getPolicyName(policyName: String): ManagedGroupPolicyName = {
     policyName match {
       case "member" => ManagedGroupService.memberPolicyName
       case "admin" => ManagedGroupService.adminPolicyName
-      case _ => throw new WorkbenchExceptionWithErrorReport(ErrorReport(StatusCodes.NotFound, "Policy name for managed groups must be one of: [\"admin\", \"member\"]"))
+      case "admin-notifiers" => ManagedGroupService.adminNotifierPolicyName
```

Comment:
should use the constants defined above instead of repeating the strings.  I think surrounding the val name in back ticks lets it be used in a case statement. e.g.
```case `adminNotifierValue` => ...```